config/example_config.yml

```diff
@@ -40,27 +40,27 @@ Data:
         # time per pixel
         use_peak_times: False
 
-        # Optional list of tel_type strings or null. Default if null: ['LST']
+        # Optional list of tel_type strings or null. Default if null: ['LST:LSTCam']
         # Valid tel_types:
-        #   - 'LST': LST Telescope and Camera
-        #   - 'MSTF': MST Telescope with FlashCAM Camera
-        #   - 'MSTN': MST Telescope with NectarCAM Camera
-        #   - 'MSTS': SCT Telescope and Camera
-        #   - 'SST1': SST-1M Telescope with DigiCam Camera
-        #   - 'SSTA': SST-2M ASTRI Telescope and Camera
-        #   - 'SSTC': SST-2M GCT Telescope with CHEC Camera
-        #   - 'VTS': VERITAS Telescope and Camera
+        #   - 'LST:LSTCam': LST Telescope and Camera
+        #   - 'MST:FlashCam': MST Telescope with FlashCAM Camera
+        #   - 'MST:NectarCam': MST Telescope with NectarCAM Camera
+        #   - 'SCT:SCTCam': SCT Telescope and Camera
+        #   - 'SST:DigiCam': SST-1M Telescope with DigiCam Camera
+        #   - 'SST:ASTRICam': SST-2M ASTRI Telescope and Camera
+        #   - 'SST:CHEC': SST-2M GCT Telescope with CHEC Camera
+        #   - 'MST:VERITAS': VERITAS Telescope and Camera
         # List of telescope types to load.
-        selected_tel_types: ['LST']
-
-        # Optional dictionary with string keys and list of integer values,
+        selected_tel_types: ['LST:LSTCam']
+        
+            # Optional dictionary with string keys and list of integer values,
```

Comment:
Looks like an extra indent here. Consider running the config file through yamllint if you haven't already to catch any style issues.

Reply:
Checked with a linter and fixed in a278272d4ed21f6a3f67fbcfbc9bf3e021ca1bca.